stories/DataTable/expandable/groupBy.stories.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import * as _ from 'lodash-es';
+
+import DataTable from '../../../src/DataTable/DataTable';
+import Row from '../../../src/DataTable/TableRow';
+import tableDataItems from '../../constants/sampleMovieData';
+
+const ExpandedComponent = ({ data, rowProps }) => {
```

Comment:
I would move this inside the library, so the user will have to pass GroupByKey and this component is used as default  `expandableRowsComponent` (if the user wants to override it, then he/she will pass his own `expandableRowsComponent` property).